PREINSTALL.md

```diff
@@ -0,0 +1,42 @@
+Use this extension to synchronize documents from a Cloud Firestore collection to a Meilisearch index. This allows you to use a full text search in your Cloud Firestore documents.
+
+This extension listens to each creation, update or deletion of your documents to keep them sync with your Meilisearch index, so the data in Meilisearch is a mirror of your content in Cloud Firestore. You can then run queries on this mirrored dataset.
+
+Note that this extension only listens for changes to _documents_ in a specific collection, but not changes in any _subcollection_. However, you can install additional instances of this extension to listen to other collections in your Firestore database.
+
+#### Additional setup
+
+Before installing this extension, you'll need to:
+
+- [Set up Cloud Firestore in your Firebase project.](https://firebase.google.com/docs/firestore/quickstart)
+- Run Meilisearch instance. There are many easy ways [to download and run a Meilisearch instance.](https://docs.meilisearch.com/learn/getting_started/installation.html#download-and-launch)
+
+#### Data import format
+
+Documents indexed in Meilisearch must have a primary key and a [unique identifier](https://docs.meilisearch.com/learn/core_concepts/documents.html#primary-field), this extension will use the default field provided for this purpose in Firestore: `Document ID` which will be renamed in `_firestore_id` and used as [primary key] and [document identifier](https://docs.meilisearch.com/learn/core_concepts/documents.html#document-id). If your documents have an `id` field, it will be added to your documents without being your primary key.
+
+**Important:**  If your document have a field `_firestore_id` it will be ignored.
+
+[GeoSearch](https://docs.meilisearch.com/reference/features/geosearch.html#geosearch) has a specific format in Meilisearch, if a `GeoPoint` from Firestore with the name `_geo` is found the field `latitude` is renamed to `lat` and the field `longitude` to `lng`.
```

Comment:
There is a specific object in `Firestore` for geo search but the two preconfigured fields in this object called `GeoPoint` are `latitude` and `longitude`. If a user wants to use geo search in Meilisearch via his objects in his Firestore he has to name his object with the prefix `_geo`.
That's whats I want to mean but not sure is clear ...

Reply:
So the Firestore version of:
```
  "_geo": {
    "lat": 0.0,
    "lng": 0.0
  }
```
is:
```
  "GeoPoint": {
    "latitude": 0.0,
    "longitude": 0.0
  }
```

If the user wants to use geosearch in Meilisearch via their objects in Firestore, they need to name this name `GeoPoint` object `_geoGeoPoint`?

Reply:
Almost the Firestore version is more like:
```
  "_geo": GeoPoint: {
    "latitude": 0.0,
    "longitude": 0.0
  }
```
And you had to name it "_geo" if you want it to be adapted to `Meilisearch` otherwise it will be ignored.